Makefile

```diff
@@ -56,8 +56,11 @@ website/build-local:
 .PHONY: license-check
 license-check:
 	go mod vendor
-	licensei check
-	licensei header
+	licensei cache --debug
+	licensei check --debug
+	licensei header --debug
```

Comment:
I've put those in debug mode to see more details about what licensei is doing.

In most cases, it is not too verbose.